scene/gui/text_edit.h

```diff
@@ -474,6 +478,9 @@ class TextEdit : public Control {
 	void set_line_as_marked(int p_line, bool p_marked);
 	void set_line_as_breakpoint(int p_line, bool p_breakpoint);
 	bool is_line_set_as_breakpoint(int p_line) const;
+	void set_line_as_executing(int p_line);
+	void clear_line_as_executing();
```

Comment:
`set_executing_line` and `clear_executing_line` sounds clearer as there can only be one.

Reply:
Renamed.